src/background/clipboard.js

```diff
@@ -0,0 +1,23 @@
+const CLIPBOARD_CLEAR_DELAY = 1 * 60 * 1000; // 1 minute
+
+let clearClipboardTimer = null;
+
+export async function copyToClipboard(navigatorClipboard, field, toCopy) {
```

Comment:
Wrestling a bit with getting a mock injected for the global `navigator.clipboard`. I just punted and kind of inject the dependency in params.

Reply:
this is a pattern we've followed if the "global" cannot be set or replaced.

I think I'd prefer for `navigatorClipboard` to be the last argument and default to `navigator.clipboard`.